iOSNativeSwiftPackageManagerTemplate/install.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
```

Comment:
The only interesting file in this PR (the rest is a copy of the iOSNativeSwiftTemplate).
We read from package.json what SPM repo / branch we should be pointing to and modify the xcode project file accordingly.